db/migrate/20231114144505_make_indexes_unique_for_standup_meeting_group_users.rb

```diff
@@ -0,0 +1,21 @@
+class MakeIndexesUniqueForStandupMeetingGroupUsers < ActiveRecord::Migration[7.0]
```

Comment:
All of this needs to be put in a separate PR.
1. We add unique constraint at model level
2. We run rake task to make sure everything is uniq at db level

Then in second PR
1. we add uniq index because it is gaurenteed to work now

Reply:
sounds good! thank you